README.md

```diff
@@ -89,6 +89,12 @@ The landing URI is where the middleware redirects the user when the
 authentication process is complete. This could just be back to the
 index page, or it could be to the user's account page.
 
+* `:basic-auth?`
+
+This is an optional parameter, which defaults to false.
+If set to true, it includes the client-id and secret as a header
+`Authorization: Basic base64(id:secret)` as recommended by [the specification](https://tools.ietf.org/html/rfc6749#section-2.3.1)
```

Comment:
Can you change this to:

    ... [the specification][].

    [the specification]: https://tools.ietf.org/html/rfc6749#section-2.3.1